Makefile

```diff
@@ -90,31 +90,38 @@ quality: clean
 quality-no-tests:
 	mvn -Dmaven.test.skip=true clean install -Pquality -Dmaven.test.skip=true
 
-STYX_HOME := $(CURRENT_DIR)/distribution/target/styx
-STYX_BUILD_ARTIFACT = $(shell find  distribution/target -name "styx.zip")
-STYX_DEPLOY_FOLDER := $(CURRENT_DIR)/distribution/target/styx
+STYX_BUILD_ARTIFACT = $(shell find  distribution/target -name "styx*.zip")
+STYX_HOME = $(CURRENT_DIR)/distribution/target/styx/styx
+CONFIG_ROOT := $(STYX_HOME)/conf/env-$(STACK)
 
 ## Compile and create styx.zip then unzip into a directory defined by STYX_HOME
 release-styx: release-no-tests
-	unzip -oq ${STYX_BUILD_ARTIFACT} -d ${STYX_HOME}
+	unzip -oq ${STYX_BUILD_ARTIFACT} -d $(dir ${STYX_HOME})
+	mv `find $(CURRENT_DIR)/distribution/target/styx -type d -name "styx-*"` ${STYX_HOME}
 
 ## Stops running netty-based origins (i.e. the origins started by start-origins)
 stop-origins:
 	(support/origins-starter-app/bin/styx-originsstopper.sh 2>&1)
 
 ## Starts netty-based origins (release-styx needs to have been run first)
 start-origins: stop-origins
-	(support/origins-starter-app/bin/styx-originsstarter.sh $(STYX_HOME) $(STYX_DEPLOY_FOLDER)/conf/$(STACK).yml &)
+	(support/origins-starter-app/bin/styx-originsstarter.sh $(STYX_HOME) $(CONFIG_ROOT)/styx-config.yml &)
```

Comment:
When running clean, start-origins fails. Can we make release-styx as prerequisite?